starter-code/styles/modules.css

```diff
@@ -9,12 +9,23 @@
   padding: 0 0 .2em 1em;
 }
 
-.icon-menu {
-  /* TODO: Add styles for the icon-menu class (whether it be the
-      hamburger font icon or an image of your choosing). It should be
-      bigger, and off to the right. */
+@media (max-width: 960px) {
+  .icon-menu {
+    /* TODO:DONE: Add styles for the icon-menu class (whether it be the
+        hamburger font icon or an image of your choosing). It should be
+        bigger, and off to the right. */
+    position: fixed;
+    top: 0;
+    right: 0;
```

Comment:
This is one way of moving the element to the right of the page. Another way (and probably the best way for this specific assignment (given that the .heading-group h1 is floated left) is to `float: right`.